### ISSUE TITLE
[2.12.2] pas d'accès à la page d'édition des permissions / CRUVED

### ISSUE THREAD

**Issue by Splendens** [PnX-SI/GeoNature#2523]
Bonjour, 
En étant identifié avec un utilisateur qui a les droits d'administrateur, j'essaye d'éditer le CRUVED d'un rôle à partir de l'interface _Permissions_. 

J'ai bien accès à la page de la _liste des rôles_, ainsi qu'à l'édition des "autres permissions". Par contre, impossible d'avoir accès à la page d'édition du CRUVED. La page charge dans le vide pendant quelques temps, avant de renvoyer une erreur 502.

Quelqu'un aurait une idée de ce qu'il se passe ? Je ne sais pas où chercher, il n'y a rien dans les logs de GeoNature.
Merci d'avance !

**Comment by bouttier** [comment]
Bonjour,
Erreur 502 = timeout (+ de 30 secondes). Je suspecte donc un problème de scalabilité lié à un grand nombre d’utilisateurs ou de groupes sur l’instance ; est-ce le cas ?

**Comment by bouttier** [comment]
Normalement réglé par #2530

**Comment by Splendens** [comment]
Effectivement, j'aurais pu préciser que je suis une instance avec quasi 5000 rôles enregistrés .

Merci pour la correction, elle règle bien le problème !
